test/unit/UnitTestEconomicStrategy.ts

```diff
@@ -12,13 +12,15 @@ import {
 import { EconomicStrategyStatus } from '../../src/Enum';
 import { ITxRequest } from '../../src/Types';
 
+// tslint:disable-next-line:no-big-function
 describe('Economic Strategy Tests', () => {
   let economicStrategyManager: IEconomicStrategyManager;
+  const MWei = new BigNumber(10000000);
```

Comment:
This is 10 mwei, isn't it? The name `MWei` might be misleading.

Reply:
Ow. That's true!